diff --git a/node_modules/react-navigation/src/index.js b/node_modules/react-navigation/src/index.js
index 3e1b4ae..8a5379c 100644
--- a/node_modules/react-navigation/src/index.js
+++ b/node_modules/react-navigation/src/index.js
@@ -1,29 +1,2 @@
 export * from '@react-navigation/core';
 export * from '@react-navigation/native';
-
-// Export each item individually so that they can be evaluated lazily
-// https://babeljs.io/docs/en/babel-plugin-transform-modules-commonjs#lazy
-export {
-  createNavigationContainer,
-  createStackNavigator,
-  createBottomTabNavigator,
-  createMaterialTopTabNavigator,
-  createDrawerNavigator,
-  StackGestureContext,
-  DrawerGestureContext,
-  DrawerRouter,
-  DrawerActions,
-  Transitioner,
-  StackView,
-  StackViewCard,
-  StackViewTransitionConfigs,
-  Header,
-  HeaderTitle,
-  HeaderBackButton,
-  HeaderStyleInterpolator,
-  DrawerView,
-  DrawerItems,
-  DrawerSidebar,
-  BottomTabBar,
-  MaterialTopTabBar,
-} from './deprecations';
